frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/vms/VmListModel.java

```diff
@@ -787,6 +788,28 @@ private void edit() {
 
     }
 
+    private void confirmBiosTypeUpdate(){
+        VM vm = getSelectedItem();
+        UnitVmModel model = (UnitVmModel) getWindow();
+        if (vm == null || model.getIsNew()) {
+            preSave();
+        } else {
+            BiosType newBiosType = model.getBiosType().getSelectedItem();
+            BiosType oldBiosType =  vm.getBiosType();
+            if (oldBiosType.equals(BiosType.Q35_SECURE_BOOT) && !newBiosType.equals(BiosType.Q35_SECURE_BOOT) ||
+                oldBiosType.equals(BiosType.Q35_OVMF) && newBiosType.getValue() < BiosType.Q35_OVMF.getValue()) {
```

Comment:
This comparison looks hacky. Can we just write `!newBiosType().isOvmf()`?

Reply:
Yes, of course, You are right.